vertx-template-engines/vertx-web-templ-pebble/src/main/java/io/vertx/ext/web/templ/impl/PebbleVertxLoader.java

```diff
@@ -29,7 +29,7 @@
  *
  * @author Paulo Lopes
  */
-class PebbleVertxLoader implements Loader<String> {
+public class PebbleVertxLoader implements Loader<String> {
```

Comment:
Depends on how you use to interpret the impl package. If this is meant to be "implementations of some interfaces" I'd keep it there. If you rather see it as a "do not use classes from this package directly" then it should be moved out.
What is the most vertx-way? :)